src/pages/MySketches/index.jsx

```diff
@@ -1,20 +1,92 @@
+import { useState, useEffect } from "react";
+
 import NavBar from "../../components/NavBar";
 import SketchCard from "../../components/SketchCard";
+import Button from "../../components/Button";
 
 const MySketches = () => {
+  const [sketches, setSketches] = useState([]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(null);
+
+  const onPrevButtonClick = () => {
+    setCurrentPage((currentPage) => currentPage - 1);
+  };
+
+  const onNextButtonClick = () => {
+    setCurrentPage((currentPage) => currentPage + 1);
+  };
+
+  const userId = sessionStorage.getItem("userEmail");
+
+  const fetchSketches = async () => {
+    try {
+      const response = await fetch(
+        `http://localhost:3000/users/${userId}/sketches?per_page=6&page=${currentPage}`,
+      );
+      const sketchesData = await response.json();
+
+      setTotalPages(sketchesData.sketches.totalPages);
+      setSketches(sketchesData.sketches.list);
+    } catch (error) {
+      console.error("Failed to fetch sketches", error.message);
+    }
+  };
+
+  useEffect(() => {
+    fetchSketches();
+  }, [currentPage]);
+
   return (
     <div className="flex flex-col h-screen">
       <NavBar />
       <main className="flex-grow grid grid-cols-3 grid-rows-2 gap-4 p-4">
-        {Array(6)
-          .fill(null)
-          .map((_, index) => (
-            <SketchCard key={index} />
-          ))}
+        {sketches.length !== 0
+          ? sketches.map((sketch, index) => (
```

Comment:
```suggestion
          ? sketches.map((sketch) => (
```
`index`는 이제 안쓰고 있으니 지우셔도 될 것 같아요.